### ISSUE TITLE
New quest: Organic products in trade?

### ISSUE THREAD

**Issue by ServusWorld** [westnordost/StreetComplete#553]
# Where it shows up
At all Objects where food is in trade. Well, …

- amenity = restaurant
- amenity = café
- amenity = fast_food
- amenity = biergarten
- amenity = bar
- amenity = pub
- shop = alcohol
- shop = bakery
- shop = beverages
- shop = brewing_supplies
- shop = butcher
- shop = cheese
- shop = chocholate
- shop = coffee
- shop = confectionery
- shop = convenience
- shop = deli
- shop = dairy
- shop = farm
- shop = greengrocer
- shop = ice_cream
- shop = pasta
- shop = pastry
- shop = seafood
- shop = spices
- shop = tea

_(Maybe still more)_
… where the `organic ` tag is missing.

# What will be added
The organic tag will be filled with `organic=no`, `organic=yes` or `organic=only`

# Form
![form_organic](https://user-images.githubusercontent.com/31391499/29753758-9d6f7720-8b78-11e7-88cb-134a4f16b9cb.png)

# Other answers
When browsing through the values for the “organic” key on taginfo, I didn’t find any useful values apart from “yes”, “no” and “only”. So I don’t have any ideas for other answers.

# Icon
Orange seems to be the color for all POI quests. This one would fit into that row.
As symbol I’d suggest a leaf, I found an SVG [here](https://pixabay.com/en/leaf-icon-nature-environment-856922/)
For this draft, I colored that icon green (same shade as on the money icon), flipped it and distorted it a bit:

![icon_organic](https://user-images.githubusercontent.com/31391499/29753799-63fcc55a-8b79-11e7-89cc-1bbe3b847867.png)

**Comment by westnordost** [comment]
Needs research in which countries this is reasonable to ask. (I.e. in many countries, there is not even the concept of "organic").

Also, a problem I see with this quest in general is that it may be weird, even in a country like Austria, to be asked as a user for each and every Frittenbude, Kneipe, bakery etc. if it offers organic food. And, if they do, will they advertise it so prominently that you can see it from the outside? I can tell at least one place in my neighbourhood where it is not apparent from the outside that all their burgers are organic.

Also, speaking of organic burgers - what to tag if the burger meat is organic, but the rest (patty, salad etc) is not and/or unknown because not advertised?

**Comment by rugk** [comment]
> And, if they do, will they advertise it so prominently that you can see it from the outside?

When they are clever, they do. :laughing:

**Comment by CloCkWeRX** [comment]
> And, if they do, will they advertise it so prominently that you can see it
> from the outside?
>
> I think that limitation assumes one kind of survey - walking around with
phone in hand. I'd be really keen to explore a gentle fork of
StreetComplete focused on "I'm sitting down/standing around in line inside
of a business, what can I answer" for things like this, #104 #99 etc.

You could even call it... "EatComplete" :P

**Comment by ServusWorld** [comment]
@CloCkWeRX
That's a good idea! Maybe such POI-specific quests shouldn't be shown on the map always, but only if you are very near (distance < 15m) to it so that it can be assumed you are inside.

Moreover, I don't think the `organic` OSM tag is so strict. According to the wiki's definition
> `organic=yes`: The shop or restaurant usually offers some organic products. For example conventional supermarkets which offer a permanent product line of organic staple food fall under that category.

an organic burger meat would be enough to at least tag the POI it with `yes`

The geographic scope is still an issue, but research can be done [here](https://en.wikipedia.org/wiki/Organic_food#Economics) or [here](https://de.wikipedia.org/wiki/Bio-Lebensmittel#Bioprodukt-Anteil_am_Lebensmittelumsatz)

**Comment by rugk** [comment]
First may be maybe discuss this in a new issue, as it get's off-topic from "organic quest" now…

IMHO a fork would not be nice, (don't wanna use two apps…) better improve this app. The best thing would be if https://github.com/westnordost/StreetComplete/issues/35 is solved and then such quests could be added as "disabled by default".

**Comment by rugk** [comment]
So I opened a new issue about that discussion to keep it out of other unrelated issues. It includes a summary of the current ideas. See https://github.com/westnordost/StreetComplete/issues/558

**Comment by westnordost** [comment]
So, my concern about spamming users with this question still remains. Also, I wouldn't know any major supermarket chain in Germany that does _not_ have a range of _Bio_ (organic) products, so I don't see that it makes sense to ask (as it also requires the surveyor to go inside).

So, what I need for this quest:
1. a reasonable choice of elements (and/or overpass query with a heuristic) where it makes sense to ask and where it is expected that you do not need to go inside. You enumerated all the tags that organic _could_ be applied to. What is needed for StreetComplete is a subset of that list (see quest suggestion guideline in the wiki: No spam, Publicly accessible, Easy answer)
2. and then, if such a sublist can be found, a list of countries where the legal concept of organic products exists to show this quest only in these countries

Then, this quest would be ready to be implemented.

**Comment by kayD** [comment]
At least in my town there are bakeries and butchers (both happen to be within a supermarket building) that offer really no organic food.

**Comment by ENT8R** [comment]
But if the majority of the stores still offers at least a small range of organic products (```organic=yes```) then this quest would be spam... 3 or 4 stores selling no organic products at all is still too less compared to a city with 100-300 stores.

**Comment by matkoniecz** [comment]
> a reasonable choice of elements (and/or overpass query with a heuristic) where it makes sense to ask and where it is expected that you do not need to go inside. You enumerated all the tags that organic could be applied to. What is needed for StreetComplete is a subset of that list (see quest suggestion guideline in the wiki: No spam, Publicly accessible, Easy answer)

My feeling is nowdays nearly everything (at least in Poland) offers some "bio" products or pretends that their product are bio.

So the quest would be basically "is this place using popular types of marketing".

**Comment by westnordost** [comment]
So I think it makes sense to implement it for (small) shops that are not part of a chain. Shops that come to my mind are:
- greengrocers
- butchers
- bakeries

Though, we still need a list of countries where the concept of organic food exists (with official labels) at all, so it can be limited to that. I added the help needed label, maybe someone would like to research that.

**Comment by matkoniecz** [comment]
https://en.wikipedia.org/wiki/Organic_certification lists all members of EU, UK, USA, Canada, Japan, India, China, Cambodia, Australia (self-governing by producers)

**Comment by trougnouf** [comment]
> > a reasonable choice of elements (and/or overpass query with a heuristic) where it makes sense to ask and where it is expected that you do not need to go inside. You enumerated all the tags that organic could be applied to. What is needed for StreetComplete is a subset of that list (see quest suggestion guideline in the wiki: No spam, Publicly accessible, Easy answer)
> 
> My feeling is nowdays nearly everything (at least in Poland) offers some "bio" products or pretends that their product are bio.
> 
> So the quest would be basically "is this place using popular types of marketing".

Not at all the case in Belgium, even though it has all kinds of cultures. Most bakeries don't, restaurants typically don't and will make it easy to know from the outside menu, butcher shops are probably fairly mixed.

I think if the user feels spammed then they will disable the quest, but it's likely no less fulfilling than some basic quests like marking all neighboring houses as houses.

**Comment by OpenGreenStreet** [comment]
I would very much appreciate if the quest would be indicated in countries like Germany/European Union with clear legal regulations (see #4068)

**Comment by matkoniecz** [comment]
https://github.com/streetcomplete/StreetComplete/issues/553#issuecomment-688214714

> Though, we still need a list of countries where the concept of organic food exists (with official labels) at all, so it can be limited to that. I added the help needed label, maybe someone would like to research that.

I createdhttps://docs.google.com/spreadsheets/d/1zYNGD_UQtSzZQ7W4jWDUdDlrx-RrFuzSvv1T9Z8w3Zc/edit?usp=sharing (editing to anyone with link, hopefully it will work and not will be overrun by spam edits)

Feel free to contribute research there! This is currently the main blocker once someone(TM) will do this I (or someone else) will implement this.

BTW, marking EU countries (second column) also may be helpful, as this template will be likely used again.

**Comment by OpenGreenStreet** [comment]
Does this help?

* https://de.wikipedia.org/wiki/Verordnung_(EG)_Nr._834/2007_(Öko-Verordnung)

* https://en.wikipedia.org/wiki/EU-organic_production-regulation

**Comment by matkoniecz** [comment]
It is mostly about countries outside EU, in EU situation is quite clear with EU standarisation.

**Comment by OpenGreenStreet** [comment]
Would it be possible to introduce the function immediately for all countries with clear legal regulations (for example, for the European Union)? Later then gradually for other countries.

**Comment by matkoniecz** [comment]
Not really - as it will take far more effort if request go one by one.

In addition truth is that I personally prefer on working on other quests that have more value and will be used more often than such disabled by default quest AND I a kind of suspicious of entire "organic" products branding.

So, I would be OK with implementing it, but I am not interested in spending lot of time on research is it supposed to be allowed in a given country or not.

And yes, for example in #4066 research where it should be enabled took about 30% of work - even in quest with tricky code and interface.

**Comment by OpenGreenStreet** [comment]
Thank you for your feedback. Of course, I find that very unfortunate, but can understand your arguments well.

**Comment by mnalis** [comment]
> I think if the user feels spammed then they will disable the quest, but it's likely no less fulfilling than some basic quests like marking all neighboring houses as houses.

Also, I think the tag [organic=*](https://wiki.openstreetmap.org/wiki/Key:organic) is problematic in itself, even in countries that require (by law) specific [certification logo](https://en.wikipedia.org/wiki/Organic_certification#Certification_and_product_labeling)  labeling (and are thus, at least in theory, less susceptible to false advertising).

For example, for `organic=no`, one would need to _check each and every article_ in the shop to be convinced that **none** of them has organic certification logo (same problem is with `organic=only`, one would need to check that **all** articles have the certification logo). 

That sounds like unreasonable amount of effort; so IMHO fails ":dollar: Valuable Surveyors" / ":chipmunk: Easy answer" [StreetComplete guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md).

For places with non-packed goods (e.g.  `amenity=restaurant`, `shop=bakery`, most `shop=greengrocer` articles etc. - basically all but convenience shops/supermarkets) the situation is even more [unverifiable](https://wiki.openstreetmap.org/wiki/Verifiability), as such goods would not usually have certification logo stamped on themselves (at least I was never served a soup with certification sticker floating in it :smile:).

**Comment by OpenGreenStreet** [comment]
Sorry, yes that is understandable, but doesn't this apply equally to halal, kosher, vegan and vegetarian products?

**Comment by matkoniecz** [comment]
@mnalis Certified "organic" products (like regular carrots are not organic but mechanical...) are typically much more expensive and therefore accompanied by prominent advertisement. At least in Poland.

**Comment by OpenGreenStreet** [comment]
@mnails Thank you very much for your detailed answer.

From my point of view, in Germany it is much easier to decide whether there are organic products in a store or restaurant than to decide whether there are halal, kosher, vegan and vegetarian products.

In stores, the two seals can be found quickly and clearly:

https://commons.wikimedia.org/wiki/File:Bio-Siegel-EG-Öko-VO-Deutschland.svg

https://commons.wikimedia.org/wiki/File:Organic-Logo.svg

In restaurants, I rely on the menu. Labeling a dish as organic when it is not would be breaking the law.

**Comment by HolgerJeromin** [comment]
In Germany there are many shop chains which have organic in its name. Bio = organic
There ist even a Wikipedia list for all chains

https://de.m.wikipedia.org/wiki/Liste_von_Biosupermarktketten

No need to check all products...

**Comment by OpenGreenStreet** [comment]
I am aware that there will always be misuse. If there is a certificate and a legal regulation, however, the probability of misuse is lower in my view.

As I said, it is more difficult for me to recognize halal, kosher, vegan and vegetarian products than organic products.

Thanks to @HolgerJeromin. for the link to the supplier list.

**Comment by NariaReynhard** [comment]
Since I see there's many concerns about this quest and it has not been debated in years. Is it still relevant to try to implement it ? Maybe it would be useful to decide "do we implement it or do we close the issue and leave this quest aside ?"

If implementing it is still meaningful, then my project group and I will probably handle it within the next weeks.

Thank you in advance

**Comment by mcliquid** [comment]
A lot has certainly happened since 2017. The concerns at that time can be summarized as follows:
- How can users reliably assess whether organic products are available alongside conventional ones?
- How is “organic” defined? (EU organic, Demeter, organic-like, etc.)

However, in my opinion, the first issue has been “resolved” already as this quest has the same complexity as vegetarian, vegan, kosher, and halal quests. The second issue is more a matter of precise wording in the quest and can be refined during development/translation.
In my perspective: **Go for it!**

**Comment by OpenGreenStreet** [comment]
I see it exactly the same way and would be very grateful if there were an optional implementation: at least in Germany, there is a growing number of grocery stores that offer exclusively organic products, and the use of the term “organic” is protected by law.

Vegetarian, vegan, kosher, and halal products can be queried, so why not organic? The “organic” parameter has long been accepted in OSM.

As I said, I would very much welcome an implementation

**Comment by mnalis** [comment]
> If implementing it is still meaningful, then my project group and I will probably handle it within the next weeks.

See above https://github.com/streetcomplete/StreetComplete/issues/553#issuecomment-688214714 by westnordost, especially part:

> Though, we still need a list of **countries** where the concept of organic food exists (**with official labels**) at all, so it can be **limited to that**. I added the help needed label, maybe someone would like to research that.

(emphasis mine). So, programming the quest is smaller part of the job, much of it would be doing (and documenting) the relevant research beforehand to find out which countries are eligible, and making it clear to the user ***which*** "organic" should be tagged (i.e. only the officially issued ones, marked by shop/restaurant displaying appropriate certification logo).

---

What would be very nice IMHO to reduce confusion (e.g. self-proclaimed "organic" vs. officially-marked organic) would be (in addition to clear words) to **display the authorized official certification logos the user should look for**. 

Preferably filtered only for the country of the quest (like e.g. we display different icons for maxweight quest depending on the country), but that is likely significantly more work, so I guess more than that you've bargained for. 

So I guess adding a few most popular official logos (e.g. in EU, USA, Asia) in an `(i)info` box might be acceptable alternative.


Also see the rest of that post, i.e.: 

> So I think it makes sense to implement it for (small) shops that are not part of a chain. Shops that come to my mind are:
<snipped to keep it short, see original post for a list>

Should help, as otherwise e.g. asking it on `supermarket` would take many hours (or likely days) to solve for a single supermarket (see https://github.com/streetcomplete/StreetComplete/issues/553#issuecomment-1145299973)

**Comment by OpenGreenStreet** [comment]
Couldn't it be handled as simply as halal, kosher, vegetarian, and vegan? (Is a label or legal regulation relevant here?)

Another aspect, in my opinion, is shops and restaurants themselves: here I would ask: exclusively, partially, or predominantly organic products

(shops: in Germany, I am thinking of Denns Biomarkt, Alnatura, LPG, Bio Company, etc.)

**Comment by mcliquid** [comment]
> Though, we still need a list of **countries** where the concept of organic food exists (**with official labels**) at all, so it can be **limited to that**. I added the help needed label, maybe someone would like to research that.

This is not intended to be a complete list, but rather an expandable starting point for this quest:
* All EU countries: Regulation (EU) 2018/848 and ECC 2092/91
** DE,AT: Bio;Öko
** IE,MT: Organic
** FR,BE,LU: Biologique;Bio
** IT: Biologico
** ES: Ecológico
** PT: Biológico
** NL,BE: Biologisch
** DK: Økologisk
** SE: Ekologisk
** FI: Luomu
** EE: Mahe
** LV: Bioloģisks
** LT: Ekologiškas
** PL: Ekologiczny
** CZ: Ekologické
** SK: Ekologické
** HU: Ökológiai
** SI: Ekološki
** HR: Ekološki
** RO: Ecologic
** BG: Биологичен
** EL: Βιολογικό
* USA: NOP from USDA
* Canada: Canada Organic Regime
* Switzerland: Bio-Verordnung --> Bio;Öko
* UK: National Law --> Organic
* Japan: Japanese Agricultural Standards (JAS) for Organic --> 有機 / Yūki; JAS Organic
* China: Organic Product Certification --> 有机
* India: National Programme for Organic Production (NPOP) --> Organic
* Australia: Australian Certified Organic --> Organic
* South Korea: Korean Organic Regulations
* Brazil: Lei de Orgânicos --> Orgânico
* Mexico: Ley de Productos Orgánicos --> Orgánico
* Iceland: Lífrænt
* Turkye: Organik
* Other countries with national law: NZ,AR,CL,PE,CR,TH,VN,ID,PH,LK,TN,MA,ET,UG,ZA

Sources:
* https://directory.ifoam.bio/certification_bodies?utf8=%E2%9C%93&filter=
* https://www.fao.org/faostat/en/#country/
* https://www.uni-goettingen.de/en/international-organic-standards-/86183.html
* https://www.fibl.org/fileadmin/documents/shop/1150-organic-world-2021.pdf

**Comment by westnordost** [comment]
@NariaReynhard you are from that project IUT de Blagnac, aren't you? In summary, I would recommend you to look for something else to contribute. This quest is a rat tail of 75% research, discussions, considerations, finding a correct and precise wording and depending on outcome of research and implementation a maintenance hell (e.g. when really displaying all those organic-products labels in the app...) while the actual _code_ implementation would be done really quickly (<100 lines of code). So, I can't recommend at all to do this as part of your project.
I can offer to collect some concrete eligible things to work on (when you tell me your preferences in what you would want to concern yourself with on the technical level), for that, I recommend creating a new topic in the discussion board and pinging me by writing @westnordost and I will see what I can do.

---

Regarding this quest, it's a long thread already, so let me kind of summarize the concerns.

1. **👨‍💻 Users are no experts:** Should only be enabled in regions where organic products are a thing. I.e. there must be at least a recognized official label for these things. [mcliquid provided some research](https://github.com/streetcomplete/StreetComplete/issues/553#issuecomment-3339407250).

2. **🕓 Effort vs impact:** In a related note, it _might_ be necessary to show how the label looks like if this is not something universally recognized because _👨‍💻 Users are no experts_. However, this would probably come with a **massive** rat-tail attached in terms of effort and maintenance effort due to the plethora of labels. -> 

3. **🕵️ Publicly accessible on foot:** Must at least be disabled by default for the same reason why the diet-quests are disabled by default

4. **💤 No spam:** To ask for e.g. every store that sells food or every restaurant would be massively spammy. The quest must be limited to those types of POIs where there is a reasonable chance that they sell organic products.

5. **🐿️ Easy answer:** For certain types of POIs, a quick an easy answer is not possible: basically any 🛒 store with shelves where you browse yourself instead of telling the clerk what you want would be very difficult to ascertain, as you'd have to check every single product to answer correctly. This is an issue for even deli, minimarkets, even most kiosks! So, not a long list of eligible POIs remain, and each will be somewhat of a pro/contra consideration (open, of course, to a rat tail of requests in the future to include this or that POI type after all...)

6. **💵 Valuable Surveyors:** It wouldn't make sense to ask for chains. E.g. organic supermarkets, they usually belong to a chain. Also, as they often have "Bio" in their name, the chance is high that it will already tagged with `organic` in the initial survey.

7. **🤔 No unanswerable quests:** / **🐿️ Easy answer:**  It is really not quite clear when to tag something as `organic=yes`. Must a comparable offer as organic products exist when compared to their usual product palette? (Like for `diet:vegetarian`?) Or would a greengrocer who has no organic vegetables at all but a few spices he offers have some organic label printed on? I guess it must be the former, the wiki hints at it. I guess there would be many edge cases where it would be difficult to decide. For example a cheese store that has many different dozens different cheese, and 3 organic ones? Yes or no?

So, in summary, this quest has issues with almost every quest guideline, give or take, and I think it would be quite difficult to navigate around all of these and still come out with a quest that actually is a net gain.
